openapi/components/requestBodies/pastoral-item.yaml

```diff
@@ -0,0 +1,129 @@
+            Each pointer will be in the format:
+            `$locator:$location:$filename`
+            The variables within the pointer have this meaning:
+            * `$locator` represents the strategy for finding the file:
+              * `store` for files uploaded to Schoolbox's storage
+              * `browse` finds files in Schoolbox's resource browser
+              * `myfile` finds files in a Schoolbox user's My Files browser
+              * `office365` finds files in an Office365 store
+            * `$location` is the location of the file within `$locator`:
+              * for `store` this is a file's unique hash
+              * for `browse` and `myfile` this is a file ID
+             * for `office365` this is the file's unique node
+            * `$filename` is the filename that the file will be given once it is uploaded to wherever it is being uploaded
+            See the [Upload a file to Schoolbox](#operation/storageAsyncUpload) endpoint for details on how to upload a file, and receive a file pointer from that upload.
+        private_attachments[]:
+          nullable: true
+          type: array
+          description: |
+            Files to attach to confidential content of the pastoral record, supplied as pointers to a file in Schoolbox.
+            Each pointer will be in the format:
+            `$locator:$location:$filename`
+            The variables within the pointer have this meaning:
+            * `$locator` represents the strategy for finding the file:
+              * `store` for files uploaded to Schoolbox's storage
+              * `browse` finds files in Schoolbox's resource browser
+              * `myfile` finds files in a Schoolbox user's My Files browser
+              * `office365` finds files in an Office365 store
+            * `$location` is the location of the file within `$locator`:
+              * for `store` this is a file's unique hash
+              * for `browse` and `myfile` this is a file ID
+             * for `office365` this is the file's unique node
+            * `$filename` is the filename that the file will be given once it is uploaded to wherever it is being uploaded
+            See the [Upload a file to Schoolbox](#operation/storageAsyncUpload) endpoint for details on how to upload a file, and receive a file pointer from that upload.
+        actions[]:
+          nullable: true
+          description: An array of IDs of pastoral action
+          type: array
+          items:
+            type: integer
+        tags:
+          nullable: true
+          description: |
+            Comma separated string of tag names.
+            If Allow Staff to Create Pastoral Care Tags config is ON, attempts to create a new tag, if not alraeady exists.
+          type: string
+        points:
+          nullable: true
+          description: Points of this pastoral record
+          type: integer
```

Comment:
we should limit this to match the front end
`Please enter a value less than or equal to 1000000.`